### PR TITLE
Create Position Status component

### PR DESCRIPTION
**Pull request by anitacaron**
Create a more generic component would be better, like `PositionStatus`. What do you think?

### CODE REVIEW

File: app/components/minister-status.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if (!endDate) {
+      // No end date set, so the mandatory is still active
+      return true;
+    } else {
+      return isDateInTheFuture(endDate);
+    }
+  }
+}
+
+function isDateInTheFuture(date) {
```
Comment (Windvis) [inline app/components/minister-status.js:17]: I think we use this elsewhere already? Maybe we can export this from a `app/utils/date.js` file or something?

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
> Create a more generic component would be better, like `PositionStatus`. What do you think?

Hmm, not sure. How would that work then? Check both the minister and mandatory status and return active when one of them is? Where do we need this exactly?

**Comment by anitacaron** [comment]
For both, we only check the end date, we can pass this to the component, and that's it.

File: app/components/minister-status.js (hunk 1 above)
Comment (anitacaron) [reply app/components/minister-status.js:17]: Yes, that's why I'm suggesting creating a generic component and pass only the end date.

Comment (Windvis) [reply app/components/minister-status.js:17]: Ok! Didn't know where we used the other one. Yea in that case renaming it to PositionStatus makes sense!

**Review by Windvis — APPROVED** [review]
Nice!

**Pull request merged**